_usage/mvu.md

```diff
@@ -0,0 +1,123 @@
+
+1. Download the zip or tar distribution file of the target Babelfish version which supports MVU from [here](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases). Extract the distribution and follow the instructions in `INSTALLING.md` to build and install new version of Babelfish in the location represented by (`$BABELFISH_HOME_NEW`), and initialize the `data` directory for the new engine version (`$BABELFISH_DATA_NEW`). 
+
+ *Note* that the target Babelfish database should be empty and should not already have the extensions installed. You should skip the `Enabling extensions in the target database` step while following the `INSTALLING.md` instructions for the target Babelfish installation.
+
+2. Stop the source and target servers (if running), with the following `pg_ctl` commands:
+
+- To stop the source server:
+
+`${BABELFISH_HOME_OLD}/bin/pg_ctl -D ${BABELFISH_DATA_OLD}/ stop`
+
+- To stop the target server:
+
+`${BABELFISH_HOME_NEW}/bin/pg_ctl -D ${BABELFISH_DATA_NEW}/ stop`
+
+3. Perform a major version upgrade to upgrade the Postgres server with the `pg_upgrade` utility. No special handling is needed to use pg_upgrade with Babelfish:
+
+```sql
+sudo su - postgres
+mkdir ~/upgrade
+cd ~/upgrade
+${BABELFISH_HOME_NEW}/bin/pg_upgrade -U $USER -b ${BABELFISH_HOME_OLD}/bin \
+                                       -B ${BABELFISH_HOME_NEW}/bin \
+                                       -d ${BABELFISH_DATA_OLD} \
+                                       -D ${BABELFISH_DATA_NEW} \
+                                       -p 5432 -P 5433 -j $JOBS --verbose
+```
+
+Refer to the [pg_upgrade documentation](https://www.postgresql.org/docs/current/pgupgrade.html) for detailed information about using `pg_upgrade`. 
+
+In the event of a failure, the new `data` directory (`$BABELFISH_DATA_NEW`) might have issues, but the old `data` directory (`$BABELFISH_DATA_OLD`) will remain intact. To retry the upgrade, you can remove the new `data` directory  and re-initialize it again. Follow the instructions in `Initiating the Data directory` from `INSTALLING.md`.
+
+After the upgrade is successfully completed, start the new server with `pg_ctl`:
+
+`${BABELFISH_HOME_NEW}/bin/pg_ctl -D ${BABELFISH_DATA_NEW} -l logfile start`
+
+To update the Babelfish extensions, connect to the PostgreSQL endpoint of the Babelfish database and execute the following statements from a single session in this order:
+
+```sql
+ALTER EXTENSION babelfishpg_common UPDATE;
+ALTER EXTENSION babelfishpg_tsql UPDATE;
+ALTER EXTENSION babelfishpg_tds UPDATE;
+```
+
+Some Babelfish specific GUCs/settings are considered critical and their values must be the same before and after a major version upgrade. Changing any of the parameter values during the upgrade might cause inconsistent database behavior or even data corruption. Therefore, it is mandatory that the following Babelfish related parameters are set to the same values after upgrade as they were before upgrade.
```

Comment:
I have added one additional bullet point (point number 3 [here](https://quip-amazon.com/aIOnA7jkumFo/Steps-to-perform-Babelfish-MVU-Open-Source-Ubuntu)) before upgrade for the user to get the initial values of the following GUCs from the source server.

Reply:
I've moved that content into a 'Prerequisites' section - please check the updated doc.